gremlin-dotnet/src/Gremlin.Net/Driver/Messages/ResponseResult.cs

```diff
@@ -23,13 +23,14 @@
 
 using System.Collections.Generic;
 using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
 
 namespace Gremlin.Net.Driver.Messages
 {
     internal class ResponseResult<T>
     {
         [JsonProperty(PropertyName = "data")]
-        public List<T> Data { get; set; }
+        public JToken Data { get; set; }
```

Comment:
You can completely remove the type parameter `T` from the class when you change the type here to `JToken`.